lib/tasks/content.rake

```diff
@@ -1,7 +1,15 @@
 namespace :content do
   desc "Exports results link content from en.yml to a csv"
-  task "export_results_to_csv" => :environment do
+  task export_results_to_csv: :environment do
     csv_data = ContentExporter.generate_results_link_csv
     File.open("tmp/results_links.csv", "w") { |file| file.write(csv_data) }
   end
+
+  desc "Imports results link content from en.yml to a csv"
+  task :import_locale_links, [:file_path] => :environment do |_, args|
+    new_result_links = ContentImporter.import_results_links(args.fetch(:file_path))
+    ContentImporter.overwrite_locale_links("config/locales/en.yml", new_result_links)
+  rescue KeyError
+    puts "Please provoide a file path"
```

Comment:
```suggestion
    puts "Please provide a file path"
```

Reply:
Urgh me and my stupid spelling, I really need to get a spellcheck for vscode and vim.

Thanks for correcting those!

Reply:
Pro-tip: If you look at the "files changed" tab, there's a button on each suggestion to "Add suggestion to batch" so you only have to commit one of the same kind of change.